en/releases/release-1.1.7.md

```diff
@@ -0,0 +1,34 @@
+---
+title: TiDB Operator 1.1.7 Release Notes
+---
+
+# TiDB Operator 1.1.7 Release Notes
+
+Release date: November 13, 2020
+
+TiDB Operator version: 1.1.7
+
+## Compatibility Changes
+
+- The behavior of `prometheus.spec.config.commandOptions` has changed. Any duplicated flags must be removed or Prometheus will fail to start ([#3390](https://github.com/pingcap/tidb-operator/pull/3390), [@mightyguava](https://github.com/mightyguava))
+
+## New Features
+
+- Support `spec.toolImage` for the `Backup` and `Restore` CR with BR to define the image used to provide the BR binary executables. Defaults to `pingcap/br:${tikv_version}` ([#3471](https://github.com/pingcap/tidb-operator/pull/3471), [@namco1992](https://github.com/namco1992))
+- Add `spec.tidb.storageVolumes`, `spec.tikv.storageVolumes`, and `spec.pd.storageVolumes` to support mounting multiple PVs for TiDB, TiKV, and PD ([#3425](https://github.com/pingcap/tidb-operator/pull/3425) [#3444](https://github.com/pingcap/tidb-operator/pull/3444), [@mikechengwei](https://github.com/mikechengwei))
+- Add `spec.tidb.readinessProbe` config to support requesting `http://127.0.0.0:10080/status` for TiDB's readiness probe, TiDB version >= v4.0.9 required ([#3438](https://github.com/pingcap/tidb-operator/pull/3438), [@july2993](https://github.com/july2993))
+- Support the case where the advanced StatefulSet controller is enabled during PD leader transfer ([#3395](https://github.com/pingcap/tidb-operator/pull/3395), [@tangwz](https://github.com/tangwz))
```

Comment:
```suggestion
- Support enabling the advanced StatefulSet controller during PD leader transfer ([#3395](https://github.com/pingcap/tidb-operator/pull/3395), [@tangwz](https://github.com/tangwz))
```
The Chinese and English notes are a little inconsistent. I'm not sure which one is correct.

Reply:
the PR is https://github.com/pingcap/tidb-operator/pull/3395, with title "fix: PD leader transfer does not consider the case where advanced statefulset controller enabled. #3395 "
Then the release note may be "support PD leader transfer with advanced StatefulSet enabled"

Reply:
Changed according to the fixed zh version, PTAL @ran-huang